src/commands/codepush/lib/update-contents-tasks/zip.ts

```diff
@@ -33,20 +34,24 @@ export default function zip(updateContentsPath: string): Promise<string> {
       });
     });
 
-    const packagePath: string = path.join(process.cwd(), generateRandomFilename(15) + ".zip");
-    let zip = new JsZip();
+    var packagePath: string = path.join(process.cwd(), generateRandomFilename(15) + ".zip");
```

Comment:
Use `let`, not `var`. Unless it can be `const`.

Reply:
my bad, fixed, thanks